packages/first-boot-wizard/Makefile

```diff
@@ -24,7 +24,6 @@ endef
 
 define Package/$(PKG_NAME)/install
 	$(CP) ./files/* $(1)/
-	@chmod a+x $(1)/etc/init.d/firstbootwizard
```

Comment:
I don't think this is related with this PR

Reply:
To correctly support the "files structure" permissions of files need to be already as they have to be in the target. That's why I changed the permissions of multiple files and removed the now unneded step in the makefile